src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/rest/LineComment.java

```diff
@@ -44,4 +46,36 @@ public LineComment(int line, String message) {
         this.message = message;
         this.line = line;
     }
+
+    /**
+     * Compares the specified object with this LineComment for equality.
+     *
+     * @param o object to be compared for equality with this LineComment
+     * @return {@code true} if the specified object is equal to this
+     *         LineComment
+     */
+    @Override
+    public final boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if ((o == null) || !(o instanceof LineComment)) {
+            return false;
+        }
+
+        LineComment e = (LineComment)o;
+        return Objects.equals(line, e.line)
```

Comment:
Though I'm fine with bumping to java 7 _(we could even go up to 8 at this point)_ doing it just to be able to use `Objects` in a PR seems quite overkill when modern IDEs usually can generate a perfectly fine `equals` and `hashCode` method without it for you ;).
You are welcome to file a separate PR that bumps to Java 7 _(or 8)_ for better changelog tracking.